lib/src/models/ewelink_device.dart

```diff
@@ -47,5 +47,9 @@ class EwelinkDevice {
 
   factory EwelinkDevice.fromJson(Map<String, dynamic> json) =>
       _$EwelinkDeviceFromJson(json);
+
+  static List<EwelinkDevice> fromJsonList(List<dynamic> json) =>
+      json.map((device) => EwelinkDevice.fromJson(device)).toList();
```

Comment:
What would happen if Json does not contain any devices? Will .map crash or will it return empty list